contracts/utils.go

```diff
@@ -0,0 +1,88 @@
+package contracts
```

Comment:
These calls feel more like `contract_comm` or `contracts/comm` or even `vm/abicaller` as a package. These are not associated with any particular contract and are used by every contract call.

Reply:
i like the idea of `vm/abicaller` or `vm/something` idea. There's already a `vm/runtime` that provides a simplified interface to the evm (we use it on mycelo).

But i'd like to first get https://github.com/celo-org/celo-blockchain/pull/1468 merged if that's possible.